### ISSUE TITLE
Language selection

### ISSUE THREAD

**Issue by SGHphoto** [PabloF7/noter#3]
I downloaded the 0.3 appimage version, it defaults to spanish on my computer. I m running Manjaro 18.04 KDE and I use a Icelandic keyboard (if that has anything to do with this error). I cant find how to switch language.

**Comment by doquen** [comment]
Hi! your language would be english right? what is your locale code? en_US, en_EN? I'll look at this this days.

**Comment by SGHphoto** [comment]
This is the language locale settings that show up when I type the locale command in cli. I use english language in the OS, Icelandic keyboard.

LANG=en_US.UTF-8
LC_CTYPE="en_US.UTF-8"
LC_NUMERIC=is_IS.UTF-8
LC_TIME=is_IS.UTF-8
LC_COLLATE="en_US.UTF-8"
LC_MONETARY=is_IS.UTF-8
LC_MESSAGES="en_US.UTF-8"
LC_PAPER=is_IS.UTF-8
LC_NAME=is_IS.UTF-8
LC_ADDRESS=is_IS.UTF-8
LC_TELEPHONE=is_IS.UTF-8
LC_MEASUREMENT=is_IS.UTF-8
LC_IDENTIFICATION=is_IS.UTF-8
LC_ALL=

**Comment by doquen** [comment]
ok, give me a few days and i'll look into it. I'm really busy right now!

**Comment by SGHphoto** [comment]
Take your time, no rush on this from me.

**Comment by SGHphoto** [comment]
I'm not much of a programmer but I imagine the program performs a check to select the language, and from your question I guess it selects based on the locale code. So I thought can it be solved with a check that would be like, if locale = spanish then select spanish else select english. Dunno if that would work and it may be a ugly hack, but since I brought this up I thought I should mention it.

**Comment by doquen** [comment]
Yes, i actually do something similar. But i think i actually check for english and if not found defaults to spanish. I just have to change the default to english.
